src/io/itk-dicom/dicom.cpp

```diff
@@ -135,24 +231,43 @@ const json import(FileNamesContainer &files) {
   // Does this affect series organization?
   seriesFileNames->SetLoadPrivateTags(false);
 
-  using SeriesIdContainer = std::vector<std::string>;
+  // These are not necessarily the DICOM SeriesUIDs due to possible mangling.
+  // Use only as internal keys.
   const SeriesIdContainer &gdcmSeriesUIDs = seriesFileNames->GetSeriesUIDs();
 
-  for (auto gdcmSeriesUID : gdcmSeriesUIDs) {
-    FileNamesContainer fileNames =
-        seriesFileNames->GetFileNames(gdcmSeriesUID.c_str());
+  SeriesIndexType curSeriesIndex;
```

Comment:
I was confused by the use of `index`, just realized that was a `map`... no suggestion here, just a remark in case you had any thoughts on making that clearer.

Reply:
Yeah, I occasionally interchange index and map, since both can do lookups. But I can rename to "map" since its less overloaded than "index".